funding.json

```diff
@@ -0,0 +1,98 @@
+{
+    "$schema": "https://fundingjson.org/schema/v1.1.0.json",
+
+    "version": "v1.0.0",
+
+    "entity": {
+        "type": "group",
+        "role": "maintainer",
+        "name": "icalendar developer community",
+        "email": "icalendar-coc@googlegroups.com",
+        "phone": "+447726466688",
+        "description": "We support the development of the icalendar library for Python, an RFC 5545 compatible parser/generator for iCalendar files.",
+        "webpageUrl": {
+            "url": "https://icalendar.readthedocs.io/",
+            "wellKnown": "https://icalendar.readthedocs.io/.well-known/funding-manifest-urls"
+        }
+    },
+
+    "projects": [{
+        "guid": "python-icalendar",         
+        "name": "icalendar",     
+        "description": "The icalendar package is a RFC 5545 compatible parser/generator for iCalendar files.",     
+        "webpageUrl": {
+            "url": "https://icalendar.readthedocs.io/",
+            "wellKnown": "https://icalendar.readthedocs.io/.well-known/funding-manifest-urls"
+        },
+        "repositoryUrl": {
+            "url": "https://github.com/collective/icalendar/",
+            "wellKnown": "https://github.com/collective/icalendar/raw/main/.github/.well-known/funding-manifest-urls"
```

Comment:
Same as other RTD vs. GH comments, except instead of the _entity_, it's about the _project_. Let's use GH URLs only.